Makefile

```diff
@@ -39,34 +38,24 @@ endif
 .PHONY: all
 all: stagingrepo stablerepo
 
-.PHONY: clean
-clean:
```

Comment:
I'd prefer to stick with a `clean` target that removes the `gh-pages` dir.

Reply:
Done.